Resources/Prototypes/_ES/Entities/Effects/viewcone_effects.yml

```diff
@@ -0,0 +1,74 @@
+# Effects spawned when certain things happen
+# that only appear to viewcone users when out of vision.
+# TODO these would probably look better as actual animations modulating alpha
+# instead of like. rsi animations
+# also i kinda wanted like, a "sound wave propagatey" sprite to underlay the actual effecta, and i thought tg had those too, but i guess not
+# idk i dont want to sprite i t.
+
+- type: entity
+  id: ESBaseViewconeEffect
+  abstract: true
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: ESViewconeOccludable
+    inverted: true
+  - type: Sprite
+    sprite: _ES/Effects/viewcone_effects.rsi
+    drawdepth: Effects
+    noRot: true
+    color: '#FFFFFF00' # 0 alpha by default, will get overriden by setalpha overlay
+    # what the fuck? removing looping didnt exist until literally a few hours ago?
+    # and now i have to version engine again?
+    # loop: false
```

Comment:
this makes the effects look kinda wrong unless i make them despawn faster than they should since they just loop automatically lol. but once we update engine ill fix